flake.nix

```diff
@@ -148,6 +152,12 @@
         kevm-pyk = prev.poetry2nix.mkPoetryApplication {
           python = prev.python310;
           projectDir = ./kevm-pyk;
+
+          postPatch = ''
+              substituteInPlace ./src/kevm_pyk/foundry.py \
+                --replace "'forge', 'build'," "'forge', 'build', '--no-auto-detect',"
+            '';
```

Comment:
`forge` does not play nice with nix by default and throws an error for me, when trying to detect `solc`. Since we supply a pinned version via nix, we add the `--no-auto-detect` flag to make forge just use that version of `solc` (see https://github.com/foundry-rs/foundry/issues/545)